Resource/PHPCSFixer/Fixer/OpeningTag.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Hoa\Devtools\Resource\PHPCsFixer\Fixer;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\FixerInterface;
+use Symfony\CS\Tokenizer\Tokens;
+use Symfony\CS\Tokenizer\Token;
+use SplFileInfo;
+
+/**
+ * Ensure there is a new line after the <php.
+ */
+class OpeningTag extends AbstractFixer
+{
+    public function fix(SplFileInfo $file, $content)
+    {
+        $tokens = Tokens::fromCode($content);
+
+        if (!$tokens->isMonolithicPhp()) {
+            return $content;
+        }
+
+        if (!$tokens[0]->isGivenKind(T_OPEN_TAG)) {
+            return $content;
+        }
+
+        if (!$tokens[1]->isWhitespace()) {
+            $tokens[0]->setContent('<?php');
+            $tokens->insertAt(1, new Token([T_WHITESPACE, "\n\n"]));
```

Comment:
there is already a fixer doing that in the PHP-CS-Fixer: ``blankline_after_open_tag``. I suggest using it instead of reimplementing your own, especially if you don't cover your implementation with tests.

And your current implementation is flawed here IIRC: it corrupts the token collection, which will impact other fixers because of the tokenization cache.
I suggest you to write tests by using the AbstractFixerTestBase of PHP-CS-Fixer, which now prevents such corruptions.